pkg/command/plugin.go

```diff
@@ -386,6 +413,40 @@ func displayPluginListOutputSplitViewContext(availablePlugins []discovery.Discov
 	}
 }
 
+func displayInstalledPluginList(installedPlugins []cli.PluginInfo, writer io.Writer) {
+	var outputData [][]string
+
+	outputWriter := component.NewOutputWriter(writer, outputFormat, "Name", "Description", "Target", "Version", "Status" /*, "Context"*/)
+
+	for index := range installedPlugins {
+		newRow := []string{
+			installedPlugins[index].Name,
+			installedPlugins[index].Description,
+			string(installedPlugins[index].Target),
+			installedPlugins[index].Version,
+			installedPlugins[index].Status,
+		}
+		// TODO(khouzam): Fix after pre-alpha
```

Comment:
Can you clarify the fix a bit?

Reply:
The TEP mentions that when listing installed plugins we should indicate if the plugin was installed from a context.
When testing this, I ran into some issues about not always having that information available.  Also, I wanted to discuss the final output format.  So, for the sake of time, I removed that information from the output for the pre-alpha release.

I have added this info in the comment in the code.